runner/package.json

```diff
@@ -40,35 +40,24 @@
     "@hapi/catbox": "11.1.0",
     "@hapi/catbox-memory": "^5.0.1",
     "@hapi/catbox-redis": "^6.0.2",
-    "@hapi/code": "8.0.1",
     "@hapi/cookie": "^11.0.2",
     "@hapi/crumb": "^8.0.1",
     "@hapi/hapi": "^20.0.3",
     "@hapi/inert": "^6.0.5",
     "@hapi/jwt": "^2.1.0",
-    "@hapi/lab": "24.0.0",
     "@hapi/scooter": "^6.0.1",
-    "@hapi/vision": "^6.1.0",
-    "@hapi/wreck": "^17.1.0",
```

Comment:
vision and wreck are modules we use in the code.

Vision is used to render the views. Wreck is a HTTP utility. I forget if we use it so just double check that

Reply:
removed vision and wreck, and replaced them with the hapi versions